script.js

```diff
@@ -0,0 +1,42 @@
+const container = document.querySelector(".container");
+const leftArrow = document.querySelector(".left");
+const rightArrow = document.querySelector(".right");
```

Comment:
Sorry, I mess it the first time, but I wrote many times before: "use single quotes for strings in js"